lib/crypto/sig_verifier.go

```diff
@@ -1,26 +1,24 @@
 // Copyright 2021 ChainSafe Systems (ON)
 // SPDX-License-Identifier: LGPL-3.0-only
 
-package runtime
+package crypto
 
 import (
-	"fmt"
 	"sync"
 	"time"
 
 	"github.com/ChainSafe/gossamer/internal/log"
-	"github.com/ChainSafe/gossamer/lib/crypto"
-	"github.com/ChainSafe/gossamer/lib/crypto/ed25519"
-	"github.com/ChainSafe/gossamer/lib/crypto/sr25519"
-	"github.com/ethereum/go-ethereum/crypto/secp256k1"
 )
 
+// SigVerifyFunc verify an signature given a pubkey and msg
```

Comment:
nit fixing up the comment :wink: 

```suggestion
// SigVerifyFunc verifies a signature given a public key and a message
```

Reply:
This is not patched, please patch, thanks!